tests/unit/conftest.py

```diff
@@ -58,21 +69,16 @@ def normalize_string():
 
 
 def get_dialect(input_dialect=None):
-    match input_dialect:
-        case "databricks":
-            return Databricks
-        case "snowflake":
-            return Snow
-        case _:
-            return input_dialect
+    # pylint: disable=protected-access
```

Comment:
not allowed to disable linter. refactor the code. perhaps the DialectConfig is an incorrect abstraction.

Reply:
Removed